Raven.Tests.Helpers/RavenTestBase.cs

```diff
@@ -138,6 +138,24 @@ protected string NewDataPath(string prefix = null, bool forceCreateDir = false)
 			return newDataDir;
 		}
 
+        /// <summary>
+        /// Creates a new Embeddable document store.
+        /// </summary>
+        /// <param name="runInMemory">Whatever the database should run purely in memory. When running in memory, nothing is written to disk and if the server is restarted all data will be lost.<br/>Default: <b>true</b></param>
+        /// <param name="requestedStorage">What storage type to use (see: RavenDB Storage engines).<br/>Allowed values: <b>esent</b>, <b>munin</b>.<br/>Default: <b>esent</b></param>
+        /// <param name="catalog"></param>
```

Comment:
activeBundles are always compiled and present, but are switched on / off on demand.

catalog contains user custom bundles (stuff that doesn't come with RavenDB)